pkgs/top-level/all-packages.nix

```diff
@@ -16341,6 +16341,8 @@ with pkgs;
 
   spooles = callPackage ../development/libraries/science/math/spooles {};
 
+  spr = callPackage ../development/tools/spr {};
```

Comment:
```suggestion
  spr = callPackage ../development/tools/spr {
    inherit (darwin.apple_sdk.frameworks) Security;
  };
```

Reply:
Hi @SuperSandro2000!

Thanks for your help.

I have integrated your suggested change. Not sure what the preferred way of doing this is, I have just amended the commit and force-pushed the change to GitHub. Hope that's okay.

Reply:
yeah, thats fine.